CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 See [RELEASE](./RELEASE.md) for workflow instructions.
 
+## UNRELEASED
+
+### Upgrade information
+
+### Highlights
+
+### Features
+
+### Improvements
+
+* [#4965](https://github.com/spacemeshos/go-spacemesh/pull/4965) Updates to PoST to prevent races and corruption of `postdata_metadata.json`.
```

Comment:
>to prevent races and corruption of `postdata_metadata.json`

It sounds really scary. Users might wrongly assume that the post code had race conditions leading to invalid results or corrupting postdata_metadata.json. That's not the case. The race was on app shutdown path only and resulted in a crash, not an undefined behavior/wrong results/postdata_metadata.json corruption. Could you be more specific here to avoid confusion?

```suggestion
* [#4965](https://github.com/spacemeshos/go-spacemesh/pull/4965) Updates to PoST:
- prevent races when closing POST verifier on app shutdown resulting in a crash,
- prevent `postdata_metadata.json` corruptions by updating it atomically.
```